services/deployment-agent/Makefile

```diff
@@ -64,6 +64,9 @@ build build-kit build-x build-devel build-devel-kit build-devel-x: ## Builds $(A
 up: .init ${DEPLOYMENT_AGENT_CONFIG} ${TEMP_COMPOSE} ## Deploys or updates current stack "$(STACK_NAME)" using replicas=X (defaults to 1)
 	@docker stack deploy -c ${TEMP_COMPOSE} $(STACK_NAME)
 
+up-aws: .init ${DEPLOYMENT_AGENT_CONFIG} ${TEMP_COMPOSE}-aws ## Deploys or updates current stack "$(STACK_NAME)" using replicas=X (defaults to 1)
+	@docker stack deploy -c ${TEMP_COMPOSE}-aws $(STACK_NAME)
+
```

Comment:
out of curiosity, do you really need to have a specific recipe for aws for the deployer? it does not need any certificate

Reply:
The only reason is this DNS problem. I need to add it to the deployer..

Reply:
It's in my todo list to look more where this problem is coming from but for now, I choosed the easy solution with that. Sure I will be able to remove it later when this problem will be solved